mne/viz/evoked.py

```diff
@@ -558,7 +560,7 @@ def _plot_image(data, ax, this_type, picks, cmap, unit, units, scalings, times,
         if mask.all():
             t_end = ", all points masked)"
         else:
-            fraction = np.float(mask.sum()) / np.float(mask.size)
+            fraction = 1 - (np.float(mask.sum()) / np.float(mask.size))
```

Comment:
fraction = 1. - np.mean(mask)

should be enough.

Reply:
Did I ever mention I got very bad grades in math?

Reply:
while I was doing maths you were doing science...